src/waypoint.jsx

```diff
@@ -252,16 +138,18 @@ export default class Waypoint extends React.Component {
       debugLog('scrollableAncestor', this.scrollableAncestor);
     }
 
-    this.scrollEventListenerId = addEventListener(
+    this.scrollEventListenerHandle = addEventListener(
```

Comment:
a thing to consider is that folks reading this could conceivably mistake this for a `window.addEventListener` due to the shadowing. I don't think anything needs changing here, but it's something I thought of while reading through the changes.